src/WorkItemMigrator/WorkItemImport/WitClient/WitClientUtils.cs

```diff
@@ -585,7 +586,8 @@ private void CorrectImagePath(WorkItem wi, WiItem wiItem, WiRevision rev, ref st
             foreach (var att in wiItem.Revisions.SelectMany(r => r.Attachments.Where(a => a.Change == ReferenceChangeType.Added)))
             {
                 var fileName = att.FilePath.Split('\\')?.Last() ?? string.Empty;
-                if (textField.Contains(fileName))
+                var encodedFileName = HttpUtility.UrlEncode(fileName);
+                if (textField.Contains(fileName) || textField.IndexOf(encodedFileName, StringComparison.OrdinalIgnoreCase) >= 0 || textField.Contains("_thumb_" + att.AttOriginId))
```

Comment:
Seemes like a really random if case

Reply:
See the original PR here: https://github.com/solidify/jira-azuredevops-migrator/pull/327

Reply:
alright :)